CMakeLists.txt

```diff
@@ -4,6 +4,14 @@ cmake_minimum_required (VERSION 3.2)
 # Project name
 project (osqp)
 
+set(OSQP_VERSION "0.0.0" CACHE STRING "The version number of OSQP")
+if(NOT OSQP_VERSION STREQUAL "0.0.0")
+    configure_file (
+        "${PROJECT_SOURCE_DIR}/include/version.h.in"
```

Comment:
I noticed that we are keeping the un-configured `osqp_configre.h.in` in the `configure/` directory rather than the include directory, and then move it to the include directory when we configure it. I think it might be good to use the same strategy here where we keep `version.h.in` inside `configure` and then configure it into the include directory.

Reply:
Good point - I missed noticing that. Done now.